### ISSUE TITLE
Some genomes with strain names not showing

### ISSUE THREAD

**Issue by ARWattam** [PATRIC3/patric3_website#532]
This is a problem with the Pseudomonas aeruginosa Liverpool Epidemic Strain genomes.  They all have strain names (see screenshot below) but in the tables and in the breadcrumb, those names aren't visible.  Its only after you open up the metadata that you find it.
![screen shot 2015-10-28 at 7 25 15 am](https://cloud.githubusercontent.com/assets/11475316/10787203/54907c18-7d45-11e5-8d68-59a528aa7d29.png)

**Comment by mshukla1** [comment]
I did consider this when we cleaned up genome name/strain names in the August release. However, the problem in this case is that all 40 "Pseudomonas aeruginosa Liverpool Epidemic Strain" genomes have exactly the same strain label, which is "LESB58-like". 

Given that it is not a definitive stain name, but rather suggesting that it is similar to some other strain (i.e. Pseudomonas aeruginosa LESB58), I have not made it part of the genome name. 

![screen shot 2015-10-28 at 9 27 46 am](https://cloud.githubusercontent.com/assets/1236463/10791590/05182854-7d57-11e5-9831-20f63f849240.png)

**Comment by ARWattam** [comment]
But someone gave it that strain name, even if it isn't unique, right?  

Rebecca

----- Original Message -----

**Comment by mshukla1** [comment]
I am not how the strain field got populated with "LESB58-like”. It could be parsed out from the 
free-text by the metadata script, oe perhaps manual curation. 

However, in any case, I just checked biosample, bioproject and genbank records for one of the 
genomes. All of them say: 
strain	Liverpool Epidemic Strain
serovar	unknown (B58-like)
http://www.ncbi.nlm.nih.gov/biosample/SAMEA2732149 <http://www.ncbi.nlm.nih.gov/biosample/SAMEA2732149>
http://www.ncbi.nlm.nih.gov/assembly/GCA_000823965.1 <http://www.ncbi.nlm.nih.gov/assembly/GCA_000823965.1>
http://www.ncbi.nlm.nih.gov/nuccore/CCOF00000000 <http://www.ncbi.nlm.nih.gov/nuccore/CCOF00000000>

Given that, I am still inclined not to add it to the genome name. 

-M

**Comment by ARWattam** [comment]
I decided to follow this up by following the link to BioSample to see what the genome is called there.  There, its called "Pseudomonas aeruginosa; LES CF sputum CF03 isolate 31".  I checked the assembly for it, too.  There its called "LES_CF_sputum_CF03_contigs_31_a1".  So in both those places, it has a more defined name (isolate 31, and I like that it identifies that it came from the sputum of a cystic fibrosis patient ).  Imagine if you are asking the scientist who isolated it for a sample to analyze.  If you didn't get to the more specific information, that person wouldn't know which one you needed.

**Comment by mshukla1** [comment]
I have added missing metadata for 40 isolates. For example:
- disease = Cystic Fibrosis
- isolation_source = LES CF sputum, patient CF03 isolate 20 

I haven't updated the genome names though, and I am not planning to. 

As we discussed, we will likely encounter such cases more often where there are multiple genomes assigned exactly the same name. It will be a time sink to come up with unique and informative genome names in such cases [we couldn't keep up with it even in PATRIC1 and PATRIC2]. 

I think the right thing to do is to make sure we capture all necessary metadata in appropriate fields and make them easily accessible on the website. 

 -Maulik
